.jenkins/gpuarray-branch

```diff
@@ -0,0 +1 @@
+v0.6.9
```

Comment:
The master of Theano should work with the master of libgpuarray. An most of the time, the master of Theano should work with the last release of libgpuarray.

So having the daily buildbot run with the master of libgpuarray and the pr buildbot run with a fixed version is great.

merging.